src/main/java/org/datavyu/plugins/ffmpeg/FfmpegJavaMediaPlayer.java

```diff
@@ -150,6 +153,10 @@ protected void playerStop() throws MediaException {
         if (0 != rc) {
             throwMediaErrorException(rc, null);
         }
+
+        if (playerGetRate() != 1.0F){
```

Comment:
Can we just initialize the rate variable with 1f?

Reply:
I will change it since we have the same logic in the VideoState.